web/e2e/src/app.e2e-spec.ts

```diff
@@ -21,3 +21,4 @@ describe('workspace-project App', () => {
     } as logging.Entry));
   });
 });
+
```

Comment:
Did this previously lack a newline at EOF? If not, better to keep it consistent with the version in `main` and not add an extra newline (and if yes, ignore this comment)